shared/package.json

```diff
@@ -7,7 +7,7 @@
     "test:watch": "jest --watch",
     "build": "webpack --env.production",
     "start:kyma": "webpack --env.development --watch",
-    "build:npx": "echo no-op, done during bootstrap"
+    "build:npx": "ln -sf ../shared/node_modules/monaco-editor/min/vs ../backend/vs"
```

Comment:
do we need `../shared` part in `../shared/node_modules/monaco-editor/min/v`?

Reply:
Yup, when in my infinite naivety I tried to do `ln -s node_modules/monaco-editor/min/vs ../backend/vs`, it created a symlink in valid target directory, but didn't take the path into account, so it tried to link `busola/backend/node_modules/monaco...`.